package.json

```diff
@@ -72,7 +72,7 @@
     "@angular/platform-browser": "~2.4.5",
     "@angular/platform-browser-dynamic": "~2.4.5",
     "@angular/platform-server": "~2.4.5",
-    "@types/jasmine": "^2.5.41",
+    "@types/jasmine": "2.5.41",
```

Comment:
Hardcoded this version for now due to `npm run package` not completing due to [this bug](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/14579)